content/docs/documents.md

```diff
@@ -34,6 +34,19 @@ Front matter contains fields. Field names prefixed with a dollar sign ($) are on
 
 As long as a blueprint specifies a document's `view` and its `path`, including YAML front matter in a content document is optional.
 
+#### Constructors
+
+Yaml constructors can be used to load in other documents or files into fields for the document.
+
+[sourcecode:yaml]
+csv:    !g.csv('/pod/path/to.csv')         # Loads CSV file.
```

Comment:
The correct usage here is actually as follows:

```!g.csv /pod/path/to.csv``` etc.

YAML constructors don't use parenthesis to invoke the functions. I also think you can drop the comment since the result should be self-explanatory (equivalent to the `g` functions in templates).